jetty-http2/http2-http-client-transport/src/main/java/org/eclipse/jetty/http2/client/http/HttpSenderOverHTTP2.java

```diff
@@ -123,19 +138,59 @@ protected void sendContent(HttpExchange exchange, HttpContent content, Callback
         else
```

Comment:
what if there was an empty content to start with... wouldn't it be consumed initially and thus the private sendContent will never be called and endStream nor trailers can be sent?

Reply:
Added a comment about why we have this immediate `callback.succeeded()`.